packages/node_modules/@node-red/editor-client/src/js/nodes.js

```diff
@@ -1032,23 +1032,31 @@ RED.nodes = (function() {
         return {nodes:removedNodes,links:removedLinks, groups: removedGroups, junctions: removedJunctions};
     }
 
+    /**
+      * Add a Subflow to the Workspace
+      *
+      * @param {object} sf The Subflow to add.
+      * @param {boolean|undefined} createNewIds Whether to update the name.
+      */
     function addSubflow(sf, createNewIds) {
         if (createNewIds) {
-            var subflowNames = Object.keys(subflows).map(function(sfid) {
-                return subflows[sfid].name;
-            });
+            // Update the Subflow name to highlight that this is a copy
+            const subflowNames = Object.keys(subflows).map(function (sfid) {
+                return subflows[sfid].name || "";
+            }).sort();
```

Comment:
```suggestion
            const subflowNames = Object.keys(subflows).map(function (sfid) {
                return subflows[sfid].name || "";
            })
            subflowNames.sort()
```

`sort` modifies the array in place. Whilst technically it does return the array reference, it is more standard to have it on its own line.

Reply:
I didn't know this standard - I'll take note of it 😉